store/postgres/src/deployment_store.rs

```diff
@@ -1579,10 +1589,72 @@ impl DeploymentStore {
                 Ok(())
             })?;
         }
+
+        //check if all indexes are valid and recreate them
+        let namespace = site.namespace.as_str();
+        let mut conn = self.get_conn()?;
+        for table in dst.tables.values() {
+            for (index_name, create_query) in table.create_postponed_indexes() {
+                let table_name = table.name.clone();
+                // Version 1 - SQL generated with diesel
```

Comment:
I am wondering is a diesel version of the query is more maintainable compared to the raw string SQL bellow. WDYT @lutter?